api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -48,7 +46,9 @@ class WorksServiceTest
 
   describe("listOrSearchWorks") {
     it("gets records in Elasticsearch") {
-      val works = createIdentifiedWorks(count = 2)
+      val works = (1 to 2).map { _ =>
+        identifiedWork()
+      }
```

Comment:
The `identifiedWorks` function which was removed in the other PR would work here and below.

Reply:
This branch is now quite badly broken after I added some stuff back into the WorkGenerators branch; I'll rebase shortly.